### PR TITLE
Support json and xml extensions

### PR DESCRIPTION
**Pull request by kuri8ive**
Responding to the need to handle json and xml

### CODE REVIEW

File: gokart/file_processor.py [hunk 1]
```diff
@@ -2,6 +2,7 @@
 import os
 from abc import abstractmethod
 from logging import getLogger
+import xml.etree.ElementTree as ET
```
Comment (hirosassa) [inline gokart/file_processor.py]: [IMO] for unity of the codes,  I recommend `et` instead of `ET` (for example, pandas as pd)

**Comment by 6syun9** [comment]
@kuri8ive LGTM!! Thank you great pull request!

**Comment by nishiba** [comment]
LGTM

File: gokart/file_processor.py (hunk 1 above)
Comment (kuri8ive) [reply gokart/file_processor.py]: In this case, I followed [Official docs](https://docs.python.org/ja/3/library/xml.etree.elementtree.html). Thanks anyway!

**Comment by hirosassa** [comment]
LGTM!

**Pull request merged**